PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java

```diff
@@ -89,6 +90,7 @@ public void onCreate(Bundle savedInstanceState) {
         scenarioNameTextView = (TextView) findViewById(R.id.scenarioNameEditText);
         listAdapter = new ArrayAdapter<>(this, R.layout.simplerow, new ArrayList<String>());
         answers = new ArrayList<>();
+        npcImageView = findViewById(R.id.askerImageView);
```

Comment:
did not use butterknife here since changing existing code will be done by infrastucture student.